open_instruct/utils.py

```diff
@@ -681,26 +737,29 @@ def submit_beaker_eval_jobs(
         command += f" --hf_revision {hf_repo_revision}"
     if len(upload_to_hf) > 0:
         command += f" --upload_to_hf {upload_to_hf}"
+    if run_oe_eval_experiments:
+        command += " --run_oe_eval_experiments"
 
     process = subprocess.Popen(["bash", "-c", command], stdout=subprocess.PIPE, stderr=subprocess.PIPE)
     stdout, stderr = process.communicate()
 
-    logger.info(f"Beaker evaluation jobs: Stdout:\n{stdout.decode()}")
-    logger.error(f"Beaker evaluation jobs: Stderr:\n{stderr.decode()}")
-    logger.info(f"Beaker evaluation jobs: process return code: {process.returncode}")
+    print(f"Beaker evaluation jobs: Stdout:\n{stdout.decode()}")
+    print(f"Beaker evaluation jobs: Stderr:\n{stderr.decode()}")
+    print(f"Beaker evaluation jobs: process return code: {process.returncode}")
 
 
+@retry_on_exception()
```

Comment:
Adding some retry on exception stuff. Helps with random network errors like the ones @yizhongw saw.

Reply:
great!